internal/controlplane/handlers_evalstatus.go

```diff
@@ -192,6 +192,10 @@ func (s *Server) ListEvaluationHistory(
 		return nil, err
 	}
 
+	if err := s.store.Commit(tx); err != nil {
+		return nil, status.Errorf(codes.Internal, "error committing transaction: %v", err)
+	}
```

Comment:
Do we care about this error though? This is a read operation after all.

Reply:
good point, resolved

Reply:
Now comes my second question... if the commit failed, do we need an explicit rollback?

Reply:
we do a rollback via defer. Would an explicit rollback make much of a difference?

Reply:
ah, you're right. I missed that. It's probably fine then.